evidence/create_build.go

```diff
@@ -0,0 +1,128 @@
+package evidence
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-artifactory/evidence/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	coreConfig "github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"github.com/jfrog/jfrog-client-go/artifactory/services"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+type createEvidenceBuild struct {
+	createEvidenceBase
+	project string
+	build   string
+}
+
+func NewCreateEvidenceBuild(serverDetails *coreConfig.ServerDetails, predicateFilePath string, predicateType string, key string, keyId string,
```

Comment:
Optional: as the types are the same, the `string` may be put just once:
```go
func NewCreateEvidenceBuild(serverDetails *coreConfig.ServerDetails, 
	predicateFilePath, predicateType, key, keyId, project, build string) Command {
```

Reply:
done